src/compas/datastructures/volmesh/core/halfface.py

```diff
@@ -934,7 +934,7 @@ def unset_edge_attribute(self, edge, name):
         u, v = edge
         if u not in self._plane or v not in self._plane[u]:
             raise KeyError(edge)
-        key = "-".join(map(str, sorted(edge)))
+        key = ",".join(map(str, sorted(edge)))
```

Comment:
Sorry if this has already been discussed, why does halfface edge attribute key not using tuples as well?

Reply:
sorry seem to not have pushed latest vers yet :)